.github/workflows/sync_crowdin.yml

```diff
@@ -38,16 +40,15 @@ jobs:
       - name: Remove empty values from JSON
         run: |
           cd assets/i18n
-          sudo chmod 777 *
```

Comment:
Was this necessary @Ushie ?

Reply:
I remember adding this fixes GitHub action permission issue.

I have no idea why.

Reply:
> To make the script executable, I'm not the one who wrote this line so I don't know about if its necessary or not, I think you could run that locally before pushing it and it'll retain permissions

The script can execute perfectly fine, it just that for whatever reason the script can't modify the JSON file.

Reply:
> Are you sure? The permissions on git look fine for this file

It is, but in GitHub Action, it doesn't. It's so confusing.

Reply:
Can you link me to a run where it occurs?